src/main/java/hudson/plugins/tasks/TasksPublisher.java

```diff
@@ -30,19 +35,183 @@
+     * Use setters to initialize if required.
+     */
+    @DataBoundConstructor
+    public TasksPublisher() {
+        super(PLUGIN_NAME);
+    }
+
+    /**
+     * Returns the Ant file-set pattern of files to work with.
+     *
+     * @return Ant file-set pattern of files to work with
+     */
+    public String getPattern() {
+        return pattern;
+    }
+
+    /**
+     * @see {@link #getPattern()}
+     */
+    @DataBoundSetter
+    public void setPattern(String pattern) {
+        this.pattern = pattern;
+    }
+
+     /**
+     * Returns the Ant file-set pattern of files to exclude from work.
+     *
+     * @return Ant file-set pattern of files to exclude from work
+     */
+    public String getExcludePattern() {
+        return excludePattern;
+    }
+
+    /**
+     * @see {@link #getExcludePattern()}
+     */
+    @DataBoundSetter
+    public void setExcludePattern(String excludePattern) {
+        this.excludePattern = excludePattern;
+    }
+
+    /**
+     * Returns the high priority task identifiers.
+     *
+     * @return the high priority task identifiers
+     */
+    public String getHigh() {
+        return high;
+    }
+
+    /**
+     * @see {@link #getHigh()}
+     */
+    @DataBoundSetter
+    public void setHigh(String high) {
+        this.high = high;
+    }
+
+    /**
+     * Returns the normal priority task identifiers.
+     *
+     * @return the normal priority task identifiers
+     */
+    public String getNormal() {
+        return normal;
+    }
+
+    /**
+     * @see {@link #getNormal()}
+     */
+    @DataBoundSetter
+    public void setNormal(String normal) {
+        this.normal = normal;
+    }
+
+    /**
+     * Returns the low priority task identifiers.
+     *
+     * @return the low priority task identifiers
+     */
+    public String getLow() {
+        return low;
+    }
+
+    /**
+     * @see {@link #getLow()}
+     */
+    @DataBoundSetter
+    public void setLow(String low) {
+        this.low = low;
+    }
+
+    /**
+     * Returns whether case should be ignored during the scanning.
+     *
+     * @return <code>true</code> if case should be ignored during the scanning
+     */
+    public boolean getIgnoreCase() {
+        return ignoreCase;
+    }
+
+    /**
+     * @see {@link #getIgnoreCase()}
+     */
+    @DataBoundSetter
+    public void setIgnoreCase(boolean ignoreCase) {
+        this.ignoreCase = ignoreCase;
+    }
+
+    /**
+     * Returns whether the identifiers should be treated as regular expression.
+     *
+     * @return <code>true</code> if the identifiers should be treated as regular expression
+     */
+    public boolean getAsRegexp() {
+        return asRegexp;
+    }
+
+    /**
+     * @see {@link #getAsRegexp()}
+     */
+    @DataBoundSetter
+    public void setAsRegexp(boolean asRegexp) {
+        this.asRegexp = asRegexp;
+    }
+
+    @Override
+    public Action getProjectAction(final AbstractProject<?, ?> project) {
```

Comment:
Might need to use `SimpleBuildStep.LastBuildAction` to make this work for `WorkflowJob` as well.